src/drv_oculus_rift/rift-hmd-radio.c

```diff
@@ -0,0 +1,296 @@
+/*
+ * Oculus Rift CV1 Radio
+ * Copyright 2016 Philipp Zabel
+ * Copyright 2019 Jan Schmidt
+ * SPDX-License-Identifier:	BSL-1.0
+ */
+#include <errno.h>
+#include <stdint.h>
+#include <string.h>
```

Comment:
String is used, if possible to do this with char*, please do so.

Reply:
This is string.h, for strcpy and memset, not the C++ string class.